### ISSUE TITLE
about visualization about training 

### ISSUE THREAD

**Issue by amiltonwong** [bearpaw/pytorch-classification#1]
Hi, @bearpaw,

Thanks for releasing this useful package. BTW, during the training , how could I visualize the loss diagram like the Tensorboard used in Tensorflow?

THX!

**Comment by bearpaw** [comment]
Sorry that I did not integrate Tensorboard in this code. However, I write a very simple logger to save the training progress. After training, you should find the `log.txt` in the directory when you save your model (e.g., `checkpoints/cifar10/alexnet/log.txt`). The log file should look like this
```
Learning Rate	Train Loss	Valid Loss	Train Acc.	Valid Acc.	
0.100000	1.451691	1.147165	47.110000	59.760000	
0.100000	0.914918	0.928320	67.084000	68.180000	
0.100000	0.719559	0.758013	74.574000	74.240000	
0.100000	0.604210	0.751205	79.030000	75.660000	
0.100000	0.538204	0.562017	81.564000	80.490000	
0.100000	0.493776	0.669323	82.940000	77.790000	
0.100000	0.465779	0.521751	83.890000	82.960000	
0.100000	0.437683	0.601297	84.980000	79.930000	
```

Then you can use the following script to visualize and compare the training process of the different models. For example, comparing the `Valid Acc.` for densenet-100 and densenet-190,
```python
from utils import *
import matplotlib.pyplot as plt

paths = {
'densenet100':'/home/wyang/code/pytorch-classification/checkpoints/cifar10/densenet-bc-100-12/log.txt', 
'densenet190':'/home/wyang/code/pytorch-classification/checkpoints/cifar10/densenet-bc-L190-k40/log.txt', 
}

field = ['Valid Acc.']

monitor = LoggerMonitor(paths)
monitor.plot(names=field)

savefig('test.eps')
plt.show()
```
![image](https://user-images.githubusercontent.com/289501/27165834-465fe674-51c9-11e7-9d76-2c33e3cb6b6d.png)

If you are willing to integrate Tensorboard into this code, feel free to create a PR. Thank you.

**Comment by amiltonwong** [comment]
Thanks @bearpaw 
FYI, [tensorboard_logger](https://github.com/TeamHG-Memex/tensorboard_logger)
